nerdlets/groundskeeper-nerdlet/nrds.scss

```diff
@@ -0,0 +1,339 @@
+    text-align: left;
+    line-height: 19px;
+    text-transform: none;
+    font-weight: 600;
+    letter-spacing: 0px;
+  }
+  
+  button[class*="Dropdown-trigger"],
+  [class*="TextField-input"] {
+    min-width: 175px;
+    min-height: 32px;
+    justify-content: space-between;
+    box-sizing: border-box;
+    padding: 6.5px 8px 6.5px 8px; // visual centering, rather than being mathematically accurate
+    font-size: 14px;
+    position: relative;
+    line-height: 19px;
+    border: none;
+    box-shadow: inset 0 0 0 1px #d5d7d7;
+    background-position: right 8px top 56%;
+    border-radius: 2px;
+    background-color: #ffffff;
+    transition: 0.075s all ease-in-out;
+  
+    &:hover {
+      background-color: #fff;
+      box-shadow: inset 0 0 0 1px #8e9494, 0 1px 0 0 #edeeee;
+      border: none;
+    }
+  
+    &:active {
+      background-color: #fff;
+      border: none;
+      box-shadow: inset 0 0 0 1px #8e9494, 0 1px 0 0 #f4f5f5;
+    }
+  
+    &:focus {
+      background-color: #fff;
+      box-shadow: inset 0 0 0 1px #d5d7d7, 0 1px 0 0 #f4f5f5;
+      border-radius: 2px 2px 0 0;
+      border: none;
+  
+      &:after {
+        content: "";
+        width: 100%;
+        height: 2px;
+        position: absolute;
+        z-index: 100000;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background-color: #70ccd3;
+      }
+    }
+  
+    &:disabled {
+      border: 1px solid #d5d7d7;
+      background-color: #f4f5f5;
+  
+      [class*="Dropdown-label"],
+      [class*="TextField-label"] {
+        color: #8e9494;
+      }
+  
+      &:hover {
+        border: 1px solid #d5d7d7;
+        background-color: #f4f5f5;
+      }
+    }
+  }
+
+  [class*="Dropdown-trigger-content"] {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  [class*="TextField-input"] {
+    &:disabled {
+      border: none;
+      box-shadow: 0 0 0 1px #d5d7d7;
+  
+      &:hover {
+        border: none;
+        box-shadow: 0 0 0 1px #d5d7d7;
+      }
+    }
+  }
+  
+  [class*="Dropdown-content"] {
+    margin-top: -1px;
+    border-color: #d5d7d7;
+    border-radius: 0 2px 2px 2px;
+    box-shadow: 0 1px 0 #f4f5f5;
+  }
+  
+  [class*="DropdownItem"]:not([class*="DropdownItem-search"]):hover {
+    background-color: #c9f3f5;
+    box-shadow: none;
+  }
+  
+  [class*="TextField-input"]:focus {
+    border-radius: 2px;
+    box-shadow: inset 0 -2px 0 #70ccd3, inset 0 0 0 1px #8e9494, 0 1px 0 0 #edeeee;
+  }
+  
+  ::-webkit-input-placeholder {
+    /* Chrome/Opera/Safari */
+    color: #8e9494;
+  }
+  ::-moz-placeholder {
+    /* Firefox 19+ */
+    color: #8e9494;
+  }
+  :-ms-input-placeholder {
+    /* IE 10+ */
+    color: #8e9494;
+  }
+  :-moz-placeholder {
+    /* Firefox 18- */
+    color: #8e9494;
+  }
+  
+  // ==============================================================
+  // Checkboxes
+  // ==============================================================
```

Comment:
Why the custom styles on checkboxes? Can we just inherit the styles provided by the NR1 SDK?